tests/06-lb-encap.sh

```diff
@@ -0,0 +1,192 @@
+tc qdisc add dev lbtest2 clsact
+tc filter add dev lbtest2 ingress prio 1 handle 1 bpf da obj tmp_lb.o sec from-netdev
+
+docker network inspect $TEST_NET 2> /dev/null || {
+	docker network create --ipv6 --subnet ::1/112 --ipam-driver cilium --driver cilium $TEST_NET
+}
+
+docker run -dt --net=$TEST_NET --name server1 -l id.server -l server1 httpd
+docker run -dt --net=$TEST_NET --name server2 -l id.server -l server2 httpd
+docker run -dt --net=$TEST_NET --name server3 -l id.server -l server3 httpd
+docker run -dt --net=$TEST_NET --name server4 -l id.server -l server4 httpd
+docker run -dt --net=$TEST_NET --name server5 -l id.server -l server5 httpd
+docker run -dt --net=$TEST_NET --name client -l id.client tgraf/nettools
+docker run -dt --net=$TEST_NET --name misc   -l id.client borkmann/misc
+
+# FIXME IPv6 DAD period
+sleep 5
+
+CLIENT_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' client)
+CLIENT_ID=$(cilium endpoint list | grep $CLIENT_IP | awk '{ print $1}')
+
+SERVER1_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' server1)
+SERVER1_ID=$(cilium endpoint list | grep $SERVER1_IP | awk '{ print $1}')
+SERVER1_IP4=$(cilium endpoint list | grep $SERVER1_IP | awk '{ print $6}')
+
+SERVER2_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' server2)
+SERVER2_ID=$(cilium endpoint list | grep $SERVER2_IP | awk '{ print $1}')
+SERVER2_IP4=$(cilium endpoint list | grep $SERVER2_IP | awk '{ print $6}')
+
+SERVER3_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' server3)
+SERVER3_ID=$(cilium endpoint list | grep $SERVER3_IP | awk '{ print $1}')
+SERVER3_IP4=$(cilium endpoint list | grep $SERVER3_IP | awk '{ print $6}')
+
+SERVER4_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' server4)
+SERVER4_ID=$(cilium endpoint list | grep $SERVER4_IP | awk '{ print $1}')
+SERVER4_IP4=$(cilium endpoint list | grep $SERVER4_IP | awk '{ print $6}')
+
+SERVER5_IP=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.GlobalIPv6Address }}' server5)
+SERVER5_ID=$(cilium endpoint list | grep $SERVER5_IP | awk '{ print $1}')
+SERVER5_IP4=$(cilium endpoint list | grep $SERVER5_IP | awk '{ print $6}')
+
+cilium endpoint config $CLIENT_ID  | grep ConntrackLocal
+cilium endpoint config $SERVER1_ID | grep ConntrackLocal
+cilium endpoint config $SERVER2_ID | grep ConntrackLocal
+cilium endpoint config $SERVER3_ID | grep ConntrackLocal
+cilium endpoint config $SERVER4_ID | grep ConntrackLocal
+cilium endpoint config $SERVER5_ID | grep ConntrackLocal
+
+#IFACE=$(ip link | grep lxc | sed -e 's/.* \(lxc[^@]*\).*/\1/')
+#for name in $IFACE; do
+#	ethtool -k $name tso off gso off gro off
+#done
+
+cilium policy delete --all 2> /dev/null || true
+cat <<EOF | cilium -D policy import -
+[{
+    "endpointSelector": {"matchLabels":{"id.server":""}},
+    "ingress": [{
```

Comment:
Agreed. The only egress policy we could add here is CIDR. I worry a bit that if this test fails because of a CIDR bug, we won't notice that and would mistake it for an encap specific bug.